### DISCUSSION TITLE
Duda sobre entrega TP0 - Warning

### DISCUSSION THREAD

**Discussion by alengeor** [INGCOM-UNRN-P1/cursada-2023#9] [Preguntas y respuestas] [answered]
Buenas profe, grupo! Espero que estén muy bien.

Tengo una duda sobre la entrega del TP0... Lo que hice, siguiendo el readme, fue luego de instalar todo lo del comprimido, probar el compilador y modificar el archivo ficha.md.

Para comprobar el compilador, ejecuté el comando **_gcc --version_** y obtuve la respuesta esperada.

Profe, debo hacer algo más para completar el TP? Asi pasamos del warning⚠️ 😞  al tick ✔️ 😃 

![image](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/110181852/ebd2dd0e-7a48-4fd1-a065-95cb263956bb)

Pido disculpas de antemano, soy nuevito en git 👉👈 

Mil mil gracias!

**Comment by martinvilu** [comment, marked as answer]
¡Fantastico!

> Pido disculpas de antemano, soy nuevito en git 👉👈

¡Ningún problema!

La forma más rápida es ejecutar
```sh
$> gcc --version > salida.txt
```
Esto enviará el texto del comando `gcc --version` al archivo `salida.txt` que podés agregar al repositorio.

(el `>` es una “redirección” de salida, hay algunos detalles adicionales en el [apunte de bash](https://github.com/INGCOM-UNRN-P1/cursada-2023/blob/main/recursos/101-BASH-r72.pdf))

**Reply by alengeor** [reply]
Genial Martín, muchas gracias! Hoy por la tarde antes de la clase voy a hacerlo como explicas. Cualquier otra duda, consulto.

Gracias de nuevo!!

**Reply by martinvilu** [reply]
No olvides marcar como resuelta la preguna ;-)

**Reply by alengeor** [reply]
Ahi va :D
